rest/server.go

```diff
@@ -0,0 +1,66 @@
+package rest
+
+import (
+	"awesomeProject3/service"
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type container struct {
+	Getter string `json:"getter"`
+	Value  string `json:"value"`
+}
+
+type httpServer struct {
+	ssService service.SecretService
+}
+
+func NewHttpServer(sr service.SecretService) httpServer {
+	return httpServer{
+		ssService: sr,
+	}
+}
+
+func (sr *httpServer) Start() {
+	http.HandleFunc("/", sr.homePage)
+	log.Fatal(http.ListenAndServe(":10000", nil))
```

Comment:
We need graceful stop for our http server and use context and goroutines to organise that.

And what do `log.Fatal` do? Is there any consequences of using it?